rose-offline-server/src/game/storage/character.rs

```diff
@@ -84,7 +84,11 @@ impl CharacterStorage {
         self.save_character_impl(&self.info.name, true)
     }
 
-    fn save_character_impl(&self, character_name: &str, allow_overwrite: bool) -> Result<(), anyhow::Error> {
+    fn save_character_impl(
+        &self,
+        character_name: &str,
+        allow_overwrite: bool,
+    ) -> Result<(), anyhow::Error> {
```

Comment:
Seems to have been a leftover `cargo fmt` artifact, I can split this out if you'd like.